### ISSUE TITLE
Problems not listed

### ISSUE THREAD

**Issue by vatai** [undertherain/benchmarker#11]
After running benchmarker with only the `--framework` parameter set I get the following output:
```shell
$ python -m benchmarker --framework pytorch
choose a problem to run
problems supported by pytorch:
```
It seems the intent was to list the available problems for the given framework, but it doesn't work (tried it with `pytorch` and `tensorflow`).

**Comment by undertherain** [comment]
absolutely, wanted to do some meaningful feedback from CLI but never got time to implement

**Comment by undertherain** [comment]
so far we have:
pytorch - getting most love
tf - though keras, dunno how much overhead it incurs compared to more low-level fashion
theano - should probably remove.. was the first one to be added back in the days :)
chainer - will also be  remove at some time in the future T_T
mxnet
dnnl -wrapper over dnnl's internal benchmark
numpy - only for problems like matrix multiplication
cupy - same for gpu
torch - also for matmul on tensors
also there are some cblas codes not yet wrapped in python...
